<!DOCTYPE html>
<html>
	<head>
		<title>Multiple Sequence Alignment</title>
		<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css">
		<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
		<script src="http://netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>
	</head>
    <body>
	
		<div class="navbar navbar-inverse navbar-fixed-top" role="navigation">
			<div class="container">
				<div class="navbar-header">
					<button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
						<span class="sr-only">Toggle navigation</span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
					</button>
					<a class="navbar-brand" href="index.html">Phylo-Made-Easy</a>
				</div>
				<div class="navbar-collapse collapse">
					<ul class="nav navbar-nav">
						<li class="active"><a href="msa1.html">Multiple Sequence Alignment</a></li>
						<li><a href="dm1.html">Distance Matrices</a></li>
						<li class="dropdown">
							<a href="build1.html" class="dropdown-toggle" data-toggle="dropdown">Building Algorithms <b class="caret"></b></a>
							<ul class="dropdown-menu">
								<li class="dropdown-header">Distance-Based</li>
								<li><a href="build1.html">UPGMA Method</a></li>
								<li><a href="build2.html">Neighbor Joining Method</a></li>
								<li><a href="build3.html">Fitch-Margoliash Method</a></li>
								<li class="divider"></li>
								<li class="dropdown-header">Character-Based</li>
								<li><a href="build4.html">Maximum Parsimony Method</a></li>
								<li><a href="build5.html">Maximum Likelihood Method</a></li>
							</ul>
						</li>
						<li><a href="terms.html">Terms and Equations</a></li>
					</ul>
				</div><!--/.nav-collapse -->
			</div>
		</div> <br><br><br>
	
	
        <h1 class="text-center">Multiple Sequence Alignment</h1> <br>
		
		<div class="span5" style="padding-left:100px;padding-right:100px">
			<p>To create a phylogenetic tree, one must first compare the sequences of interest with each other to obtain
			a rough estimate of their genetic diversity. This is accomplished through a multiple sequence alignment. Such an alignment is beneficial
			as it calculates the frequency of nucleotides in each sequential position, which will allow one to visualize the point mutations which
			occurred over time and therefore the genetic similarity among the sequences. </p>
			<div class="row text-center">
				<img src="http://upload.wikimedia.org/wikipedia/commons/7/79/RPLP0_90_ClustalW_aln.gif" height="400" width="600">
			</div> <br>
			<p>The most common tools used by bioinformaticians to align multiple sequences are ClustalW and MUSCLE, both of which
			can be accessed by Biopython, a common module with many functions useful to bioinformatics. It is recommended to use ClustalW when aligning DNA and
			mRNA sequences, and MUSCLE when aligning protein sequences. For information on how to incorporate Biopython, ClustalW, and MUSCLE into your 
			Python program, use the	<a href="http://biopython.org/DIST/docs/tutorial/Tutorial.html">Biopython Cookbook</a>.</p> <br>
			<div class="text-center">
				<a href="dm1.html" class="btn btn-default" role="button">Next Section</a>
			</div>
    </body>
	<footer class="footer"><br><br></footer>
</html>
